src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -12,6 +12,22 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
+  <ItemGroup>
+    <EmbeddedResource Update="ConverterResources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <CustomToolNamespace>Microsoft.CodeAnalysis.Sarif</CustomToolNamespace>
+      <LastGenOutput>ConverterResources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="ConverterResources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ConverterResources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
```

Comment:
>ItemGroup [](start = 4, length = 9)

Years ago (literally), when we converted these projects to .NET Core, we removed all the .cs files. But it turns out you still need the incantation for handling .resx files. You don't notice the problem until you try to add a new resource string and you find the .Designer.cs file doesn't get updated and your build fails.

We've run into this and fixed it in other projects (see for example SdkResources.resx in Sarif.csproj. There might or might not be more of these left to fix.

Reply:
thanks for explanation, good to know!

---
In reply to: [289120150](https://github.com/microsoft/sarif-sdk/pull/1513#discussion_r289120150) [](ancestors = 289120150)